manifest.json.in

```diff
@@ -0,0 +1,27 @@
+{
+  "id": "@ID@",
+  "branch": "@BRANCH@",
+  "build-extension": true,
+  "runtime": "org.learningequality.Kolibri",
+  "runtime-version": "master",
```

Comment:
Kolibri on Flathub is branch `stable`, not `master`. This also ends up being used by the generator when running Kolibri. So I don't think this will work unless you have a hand-built Kolibri Flatpak installed.